packages/eas-cli/package.json

```diff
@@ -34,9 +36,11 @@
     "pkg-dir": "^4.2.0",
     "progress": "^2.0.3",
     "prompts": "^2.3.2",
+    "tar": "^6.0.5",
     "tslib": "^1",
     "untildify": "^4.0.0",
     "uuid": "^8.3.0",
+    "validator": "^13.1.17",
```

Comment:
As far as I can see, we're only using it for UUID validation. 
Let's use https://github.com/uuidjs/uuid#uuidvalidatestr instead.
```suggestion
```

Reply:
We also use it for validating URL: https://github.com/expo/eas-cli/blob/a28ffba4580b2e87b8c720ca124fc154980c08b2/packages/eas-cli/src/submissions/archive-source/ArchiveFileSource.ts#L264

Reply:
Hmm, maybe let's find some way that doesn't use `validator.isURL` for URL validation. Maybe https://nodejs.org/api/url.html can help us.